lib/nerves_motd/runtime.ex

```diff
@@ -33,6 +34,21 @@ defmodule NervesMOTD.Runtime.Target do
     %{started: started, loaded: loaded}
   end
 
+  @impl NervesMOTD.Runtime
+  def cpu_temperature() do
+    # Read the file /sys/class/thermal/thermal_zone0/temp. The file content is
+    # an integer in millidegree Celsius, which looks like:
+    #
+    #     39008\n
+
+    with {:ok, content} <- File.read("/sys/class/thermal/thermal_zone0/temp"),
+         {temperature, _} <- Integer.parse(content) do
+      :erlang.float_to_binary(temperature / 1000, decimals: 3)
```

Comment:
I left a previous comment to just return a number here rather than convert to a string. The idea being to format close to the code that presents the value. On formatting, my personal opinion is that only 1 digit after the decimal point is useful for Celsius.